packages/node_modules/@webex/internal-plugin-encryption/src/kms.js

```diff
@@ -661,6 +663,17 @@ const KMS = WebexPlugin.extend({
     context.ephemeralKey = originalContext.ephemeralKey;
 
     return context;
+  },
+
+  updateKmsKey(activity) {
+    activity.clientTempId = `web-client-${uuid.v4()}`;
+
+    return Promise.resolve(
+      this.webex.internal.conversation
+        .get({url: activity.target.url})
+        .then((conversation) =>
+          this.webex.internal.conversation.updateKey(conversation, null, activity).then((result) => result))
```

Comment:
To improve readability, please refactor this section based on line length and callback references:

Usable example:
```js
  .then((conversation) => this.webex.internal.conversation.updateKey(
    conversation,
    null,
    activity
  )
  .then((result) => result))
```

Reply:
Okay, Will update it.